NAMESPACE

```diff
@@ -1,5 +1,7 @@
 # Generated by roxygen2: do not edit by hand
 
+export("geomean*sd")
+export("geomean/sd")

```

Comment:
Is this a valid function name

Reply:
@pchelle Please rename the functions. You have to call the functions in quotes otherwise 
```
`geomean*sd`(c(1,2,3))
```
Not sure if export must contain the quotes as well then, but having such a function name is pretty crazy from my point of view.

Reply:
I agree. I don't even know why this is allowed